app/code/Magento/Captcha/Model/Customer/Plugin/AjaxLogin.php

```diff
@@ -35,25 +40,29 @@ class AjaxLogin
      * @param CaptchaHelper $helper
      * @param SessionManagerInterface $sessionManager
      * @param JsonFactory $resultJsonFactory
+     * @param \Magento\Framework\Serialize\SerializerInterface|null $serializer
      * @param array $formIds
+     * @throws \RuntimeException
      */
     public function __construct(
         CaptchaHelper $helper,
         SessionManagerInterface $sessionManager,
         JsonFactory $resultJsonFactory,
-        array $formIds
+        array $formIds,
+        \Magento\Framework\Serialize\SerializerInterface $serializer = null
     ) {
         $this->helper = $helper;
         $this->sessionManager = $sessionManager;
         $this->resultJsonFactory = $resultJsonFactory;
+        $this->serializer = $serializer ?: \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
Please use `Magento\Framework\Serialize\Serializer\Json` instead of `SerializerInterface`, since we can not guarantee that preference will remain Json in the future.